File: Java/Gpu.java
public class Gpu extends Komponen {
    private int vramGB;
    private double kecepatanClock;
    private OverclockLimit overclockLimit;

    public Gpu(int vramGB, double kecepatanClock, double maxOverclock, String merk, String nama) {
        super(merk, nama);
        this.vramGB = vramGB;
        this.kecepatanClock = kecepatanClock;
        this.overclockLimit = new OverclockLimit(maxOverclock);
    }

    public int getVramGB() { return vramGB; }
    public double getKecepatanClock() { return kecepatanClock; }
    public double getMaxOverclock() { return overclockLimit.getMaxOverclock(); }

    public void setKecepatanClock(double kecepatanClock) { this.kecepatanClock = kecepatanClock; }
    public void setMaxOverclock(double maxOverclock) { this.overclockLimit.setMaxOverclock(maxOverclock); }
}

File: Python/Gpu.py
from Komponen import Komponen
from OverclockLimit import OverclockLimit

class Gpu(Komponen, OverclockLimit):
    def __init__(self, vram_gb, max_overclock, merk, nama):
        Komponen.__init__(self, merk, nama)  # Inisialisasi Komponen
        OverclockLimit.__init__(self, max_overclock)  # Inisialisasi OverclockLimit
        self._vram_gb = vram_gb

    def get_vram_gb(self):
        return self._vram_gb

    def set_vram_gb(self, vram_gb):
        self._vram_gb = vram_gb